pangeo-binder/values.yaml

```diff
@@ -189,19 +193,39 @@ kube-lego:
 
 dask-gateway:
   gateway:
-    clusterManager:
-      image:
-        name: daskgateway/dask-gateway-server
-        tag: 0.7.0
-        pullPolicy: IfNotPresent
-      clusterStartTimeout: 600
-      workerStartTimeout: 600
+    prefix: "/services/dask-gateway"
+    image:
+      name: daskgateway/dask-gateway-server
+      tag: 0.7.1
+      pullPolicy: IfNotPresent
+
+    backend:
+      scheduler:
+        cores:
+          request: 0.8
+          limit: 1
+        memory:
+          request: 1G
+          limit: 2G
```

Comment:
added explicit resource requests here. many instances seem to have multiples of 2vCPUs, the default request of 1 vCPU was firing up new small nodes b/c there are always a few other default pods running that request 100m CPU etc.